packages/design-system-tailwind-preset/package.json

```diff
@@ -49,6 +49,7 @@
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
     "@types/jest": "^27.4.1",
+    "@types/node": "^16.18.54",
```

Comment:
Why is this necessary?

Reply:
This is necessary because by setting the tsconfig.packages.json types array to empty we are essentially ignoring all automatic types which included `@types/node` so now we have to set it for each individual package. We may want to revisit how our tsconfigs are setup

Reply:
This was to resolve a test failure